docs/account-management/connecting-bitbucket-and-semaphore.md

```diff
@@ -0,0 +1,122 @@
+---
+Description: This document provides instructions on how to connect your Bitbucket account to Semaphore 2.0.
+---
+
+# Connecting your Bitbucket account and Semaphore
+!!! beta "Bitbucket integration is in beta"
+    This feature is in beta. Beta features are subject to change as our team continues working on improving them further. If you experience any issues while using this feature please contact our support. 
+
+Semaphore integrates with multiple git providers. You can connect your Bitbucket account with Semaphore by:
+
+- Creating a new Semaphore account with Bitbucket.
+- Connect it to an existing Semaphore account, previously created with another git provider like GitHub.
+
+
+### Creating a new Semaphore account with Bitbucket
+
+
+1. Make sure you are logged into your Bitbucket account that you want to connect.
+2. Go to the Semaphore [signup page](https://id.semaphoreci.com/signup) and select "Log in with Bitbucket"
+3. To support single sign-on via Bitbucket, Semaphore uses [OAuth App](https://support.atlassian.com/bitbucket-cloud/docs/use-oauth-on-bitbucket-cloud/). Grant access to the Semaphore OAuth App (you can always [revoke it later](#disconnecting-your-bitbucket-account)).
+4. Create a new Semaphore organization if you are not already part of one. 
+
+
+!!! info "Using the same email address on GitHub and Bitbucket"
+    Semaphore uses an email address as a unique identifier. If you use the same email on your GitHub and Bitbucket accounts, and you have previously connected the Semaphore account with GitHub, you will not be able to create a new one with Bitbucket. 
+
+    **Instead, you have to connect your BitBucket account to your existing Semaphore profile.** 
```

Comment:
why bold this case, and not eg. steps that 100% of people need to take...

Reply:
@markoa, it is an edge case, although, it is a confusing one. If this happens, we display an error message in-app and direct it to support. Still, I thought it would be good also to have it documented somewhere. Maybe I should skip this not altogether?